pkg/controller/storagecluster/storagecluster.go

```diff
@@ -317,6 +335,19 @@ func (c *Controller) syncStorageCluster(
 		return c.deleteStorageCluster(cluster)
 	}
 
+	// If portworx has been installed by daemonset, before it's uninstalled, operator
+	// should not proceed with installation.
+	if exists, err := c.isDaemonSetPresent(portworxDaemonSetName); err != nil || exists {
+		if err != nil {
+			return err
+		}
+
+		if exists {
+			return fmt.Errorf("daemonset %s exists, please delete it first. Operator will not proceed with installation",
```

Comment:
Let's not ask users to delete the daemonset now. We should just warn them that both installation methods won't work simultaneously and that's why we cannot proceed.